pkg/twcc/twcc_test.go

```diff
@@ -255,6 +255,53 @@ func Test_feedback(t *testing.T) {
 		}
 	})
 
+	t.Run("add received small deltas", func(t *testing.T) {
+		f := newFeedback(0, 0, 0)
+		base := int64(320 * 1000)
+		deltaUS := int64(200)
+		f.setBase(5, base)
+
+		for i := int64(0); i < 5; i++ {
```

Comment:
Can probably start the loop from 1? It is re-adding the base which is probably fine. `addReceived` code will cycle through the 16-bit space in the loop here 
```
    for ; f.nextSequenceNumber != sequenceNumber; f.nextSequenceNumber++ {                                                   
        if !f.lastChunk.canAdd(rtcp.TypeTCCPacketNotReceived) {                                                              
            f.chunks = append(f.chunks, f.lastChunk.encode())                                                                
        }                                                                                                                    
        f.lastChunk.add(rtcp.TypeTCCPacketNotReceived)                                                                       
        f.sequenceNumberCount++                                                                                              
    } 
```
which will affect the sequence number count. But, that is not tested here. Can change to loop to start from `1` to avoid that.